render-ws-java-client/src/main/resources/logback.xml

```diff
@@ -7,9 +7,62 @@
             <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level [%logger] %msg%n</pattern>
         </encoder>
     </appender>
-  
-  <root level="debug">
-    <appender-ref ref="STDOUT" />
-  </root>
 
-</configuration>
\ No newline at end of file
+    <appender name="FILE-INFO" class="ch.qos.logback.core.rolling.RollingFileAppender">
```

Comment:
This is the log config that I used for the service setup. The idea was to be able to use different log files for info messages and for error messages.